### ISSUE TITLE
propel.php config:convert ignores dateTime settings

### ISSUE THREAD

**Issue by hubbd** [propelorm/Propel2#1456]
Hi all,
first of all: Thank you so very much for such a fantastic orm! I've been using it for a while now and I love it.
I think I found a bug - or I'm doing something wrong, you tell me:

propelorm is up to date per composer (~2.0dev)

The config file is propel.php. Using propel config:convert it creates a functioning config.php including database, runtime and generator sections.

But if I add a "dateTime" section as per the config documentation like so...
...
'generator' => [
            'defaultConnection' => 'myconnection',
            'connections' => ['myconnection'],
	    'dateTime' => [
				'useDateTimeClass' => false,
				'defaultTimeFormat' => "H:i:s"
			]
        ]
... the dateTime settings are completely ignored -> nothing changes in the config.php

Infact, unless I delete the config.php first, the output even reads "No change required in the current configuration file"

If I move the "dateTime"-Array anywhere else outside generator or if I misspell it, an exception is thrown.

So to summarize: It seems the config:convert script is aware of the existence of the paramaters and its validity, but doesn't include it into the config.php

Or am I getting it all wrong?

Just FYI: The reason I was looking for this in the first place: I've got a mysql table with TIME columns. When I do $Query->toArray() the fields are shown like "2018-01-28T10:00:00+00:00". How else could that be changed without changing the defaultTimeFormat?

Thank you very much!
